script/glam/export_csv

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# To be run in glam via:
+#    docker-compose run server ./manage.py import_glean_aggs
+
+set -e
+
+src_project=${SRC_PROJECT:-glam-fenix-dev}
+# TODO: glam-dev-bespoke-nonprod-dataops-mozgcp-net
```

Comment:
I don't have permissions to write into this bucket, so I'll modify the glam import script accordingly.